src/encoded/audit/biosample.py

```diff
@@ -178,15 +178,27 @@ def audit_biosample_part_of_consistency(value, system):
 # utility functions
 
 def is_part_of(term_id, part_of_term_id, ontology):
-    if 'part_of' not in ontology[term_id] or ontology[term_id]['part_of'] == []:
-        return False
-    if part_of_term_id in ontology[term_id]['part_of']:
-        return True
-    else:
-        parents = []
-        for x in ontology[term_id]['part_of']:
-            parents.append(is_part_of(x, part_of_term_id, ontology))
-        return any(parents)
+    """
+    Given the term_ids for a child and parent biosample pair as obtained from the
```

Comment:
May be good to have examples of the arguments in the doc string.  Also, I'd like to know more about them to verify the functionality.